gradle.properties

```diff
@@ -10,5 +10,8 @@ org.gradle.jvmargs=-Xmx2G -XX:MaxMetaspaceSize=1G
 # SDK
 sdkVersion=0.28.2-SNAPSHOT
 
+# codegen
+smithyKotlinCodegenVersion=0.28.2-SNAPSHOT
```

Comment:
**Suggestion**: I like the simplicity of `sdkVersion` for the runtime and by comparison `smithyKotlinCodegenVersion` seems wordy and verbose. How about `codegenVersion`?